package.json

```diff
@@ -45,6 +45,7 @@
     "anymatch": "^1.3.0",
     "async-each": "^0.1.6",
     "glob-parent": "^2.0.0",
+    "inherits": "^2.0.1",
```

Comment:
I am using chokidar in the browser using a virtual filesystem (fs-memory) so the fewer node dependencies, the better for me.